docker/Dockerfile

```diff
@@ -15,7 +15,7 @@
 #  builder-install - Build environment  #
 # ##################################### #
 
-FROM ubuntu:18.04 AS builder-install
+FROM debian:stable-slim AS builder-install
```

Comment:
i think the version of the SGX driver we install may only support ubuntu 18.04, we should check if that's right

Reply:
Wow this was supposed to be just on my private repo. Sorry to bug you with my experimenting

Reply:
no worries!